week1/project/hackyourweather/src/main.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import cityData from './city-weather.json';
+
+function MainWeather() {
+  return (
+    <div>
+      {cityData.map((data) => {
```

Comment:
You can destructure inside the constructor
```suggestion
      {cityData.map(({name, sys, weather, main, coord }) =>  {
```